### DISCUSSION TITLE
Adding Wazuh agent to 4.9.1 Dashboard

### DISCUSSION THREAD

**Discussion by AltdarkAde** [wazuh/wazuh#26573] [Q&A]
Hello Everyone,

How can i add a wazuh agent on my Windows OS? I cannot find any option for adding an agent on the new dashboard interface.

**Comment by MiguelazoDS** [comment]
Hi @AltdarkAde, 

Is this what you're looking for? 

https://documentation.wazuh.com/current/installation-guide/wazuh-agent/index.html

![image](https://github.com/user-attachments/assets/450601c1-fa6b-466b-950f-7862e3e95887)

You need to copy the link provided in the agent PowerShell.

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open it if needed
